BoscoTing/TwitchAdvisor
# Feature/count chatroom info

## Description

1. Build basic features: message_count, chatters_count and sentiment_score.
2. Show these features using streamlit.

## Testing Scope
1. Replace the name of "channel" variable with an active channel name in both data_source/chat_log.py and dashboard_info/stats.py
2. Execute $python data_source/chat_log.py to check if messages in chatroom are recorded in log/chat.log file
3. Execute $streamlit run dashboard_info/stats.py
